activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -373,6 +380,17 @@ def group!(*args) # :nodoc:
     #
     #   User.order('name DESC, email')
     #   # SELECT "users".* FROM "users" ORDER BY name DESC, email
+    #
+    # This could be a source of SQL injection, so only strings composed of plain column names and simple function(column_name) expressions with optional ASC/DESC modifiers are allowed.
+    #
+    # === arel
+    # If you need to pass in complicated expressions, that you have validated are safe for the DB, you can use Arel.
+    #   User.order(Arel.sql('end_date - start_date'))
+    #   # SELECT "users".* FROM "users" ORDER BY end_date - start_date
+    #
+    #   # Custom DB Syntax like JSON columns for Postgres are supported in this way.
```

Comment:
The other examples in this file seem to use commented out code examples for the resulting queries only.
Maybe use regular text for this as well:
```suggestion
    # Custom DB Syntax like JSON columns for Postgres are supported in this way.
```